platforms-maturity-model/v1/index.md

```diff
@@ -0,0 +1,480 @@
+
+###### Characteristics:
+
+* Before delivering any new platform feature, the team discusses how to evaluate the outcome from their work
+* Broad alignment on measures that indicate success of platform initiatives
+* A product manager or dedicated team member drives an ongoing and consistent feedback collection and analysis process
+* The organization has established metrics and goals to observe and target to indicate success.
+
+###### Examples Scenarios:
+
+* The organization has consistently tracked build times and lead time. However, now they are investing in a complete set of data around the DORA metrics allowing for a more complete picture of how velocity and stability are interacting in regards to their delivery process.
+
+##### Level 4, Optimized - Quantitative and qualitative
+
+Feedback and measurements are deeply integrated into the organization's culture. The entire organization, from top-level executives to engineers organization-wide, recognizes the value of data collection and feedback on product evolution. There is a democratization of data, where various stakeholders, including platform users and business leaders, are actively involved in identifying hypotheses for platform improvements, providing feedback during the design process, and then measuring the impact post delivery. All of these measurements are considered when planning platform initiatives.
+
+Not only are standard frameworks leveraged, but there is an understanding that measuring from multiple angles creates a more holistic picture. There is an investment in understanding how qualitative measures change as quantitative ones are improved. There is a focus on identifying leading measures which can allow anticipation of features that would support user needs, alleviate their challenges, and stay ahead of industry trends and business requirements.
+
+###### Characteristics:
+
+* Platform teams continuously seek ways to improve the metrics they watch and the way they gather data
+* Sensitivity to [Goodhart's Law](https://en.wikipedia.org/wiki/Goodhart%27s_law): "When a measure becomes a target, it ceases to be a good measure."
+* Metrics and telemetry gathered is continuously evaluated for true insight and value
+* Metric data management is well supported, such as standard platform capabilities to manage data lakes and derive insights
+* Cross-departmental collaboration is encouraged to avoid data silos and enable effective feedback cycles
+
+###### Examples Scenarios:
+
+* Over time the organization has collected data indicating a rise in build time of over 15% triggers negative developer experiences and that once triggered, even if the build time is reduced below the original time developers stay frustrated for longer. This allows the build team to set a Service Level Objective (SLO) which enables early identification and improvement before instigating the negative cycle with their users.
```

Comment:
I'm just contemplating if this should have a short conclusion as well. Currently, it ends abruptly here and we don't mention anything that the reader should do after this. 

Maybe the section `A final introductory note` can be tweaked and pushed as a conclusion which would logically close the document in a way "_okay so you read the document completely, what next? Well, remember platforms will vary from organization to organization, so make sure to evaluate the current state of your group's overall cloud native transformations. After all there's no one-size fits all!_"

Nothing very elaborate, but a few sentences to logically close the document and provide some action points.

Reply:
Agree with @techmaharaj a short conclusion or what's next/further reading could help round up the document. Will try come up with something that ties it in back to the intro, but brain is mush right now.

Reply:
Was coming around to resolve this as we now have a conclusion, however we did not add in the concept of `further reading` as you suggested @puja108 . I think this is a good idea, but I am hesitant to tackle it right now. Did you have any concrete suggestions for what to offer or is this something we could maybe do later on as a blog post etc around this paper?

Reply:
FWIW originally the Definition paper included a long list of source and reference articles, but we decided to take it out for final publication. I encouraged taking them out cause it's hard to choose the right links at the start and then it's hard to be sure they're up-to-date and working in the future.

Based on that I recommend that if we want to create a list of reference and source links - an "awesome" list of sorts for platforms - we should do it in a separate document.

Reply:
WFM, let's leave it out then and let the paper work by itself, that way it might work better anyway.